lua/main.lua

```diff
@@ -141,8 +141,7 @@ Unfortunately I can't work in normal groups. If you need me, please ask the crea
 
 			-- log this event
 			if config.bot_settings.stream_commands then
-				print(string.format('%s[%s]%s Bot was added to a normal group %s%s [%d] -> [%d]',
-					clr.blue, os.date('%X'), clr.yellow, clr.reset, msg.from.first_name, msg.from.id, msg.chat.id))
+				print(clr.blue..'['..os.date('%X')..']'..clr.yellow..' Bot was added to a normal group'..clr.reset..''..msg.from.first_name..' ['..msg.from.id..'] -> ['..msg.chat.id..']')
```

Comment:
not a big fan of this. I'd much prefer actually checking the relevant keys.

Reply:
``lua: ./lua/main.lua:144: bad argument #2 to 'format' (string expected, got table)
stack traceback:
	[C]: in function 'format'
	./lua/main.lua:144: in function <./lua/main.lua:61>
	(tail call): ?
	polling.lua:47: in main chunk
	[C]: ?``

Reply:
why don't you merge this pull request?